app_dart/test/request_handlers/github/webhook_subscription_test.dart

```diff
@@ -2587,6 +2598,255 @@ void foo() {
         await tester.post(webhook);
       });
     });
+
```

Comment:
Add a test that if:

- emergency is added
- there are no approvals
- nothing happens

... but then if an aprpoval happens (and it's a flutter-hacker)

the emergency label is processed as expected (we talked about this offline)